tests/query.rs

```diff
@@ -139,10 +139,10 @@ fn fetch_meta() {
                 playground::RetryExpectations::ExpectOut,
                 |json| -> TestResult<bool> {
                     val = json.clone();
-                    Ok(true)
+                    // Wait until the document has been indexed
+                    Ok(val["results"].as_array().unwrap().len() == 1)
```

Comment:
This test was flaky due to a race condition where the index would be created but the document created in the test had not been indexed yet, so the query would succeed but there would be no results.